src/airflow/dags/data_warehouse.py

```diff
@@ -5,7 +5,7 @@
 
 @dag(
     "Consolidated-Data-ETL",
-    schedule= "@weekly",
+    schedule= "0 0 * * *",
```

Comment:
thanks @BenjaminSsempala , running once a day sounds like a good configuration for a data warehouse.
Perhaps I need more context on this. Could this be the "job" which syncs new data from other sources? If yes, does it also consider UPDATED data?

Reply:
Hey @Baalmart 
This job merges hourly low cost and bam data plus weather data for the given time period, and also add site and device details. This is used for analysis.
The corresponding CleanUp dag cleans up the duplictes.

Reply:
thanks @BenjaminSsempala  for this!